cs/src/Connections/WebSocketStream.cs

```diff
@@ -6,519 +6,547 @@
 using System;
 using System.Diagnostics;
 using System.IO;
+using System.Net;
 using System.Net.WebSockets;
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Microsoft.VsSaaS.TunnelService
+namespace Microsoft.VsSaaS.TunnelService;
```

Comment:
Lots of changes in this file making it hard to review. Perhaps changing to file-scoped namespace may happen in a separate PR to simplify reviewing this one?

Reply:
For future reference, GitHub can hide whitespace differences:
![image](https://user-images.githubusercontent.com/13093042/184263872-6f297572-f5f4-4fe2-a735-1bb01389acbc.png)

But I closed this PR anyway.

Reply:
Thanks for the tip on the whitespace diffing!